### ISSUE TITLE
Design 5

### ISSUE THREAD

**Issue by rishikeshreddyakkireddy** [rishikeshreddyakkireddy/designtodevelopment#5]
Develop this design using your preferred web technologies. Try to approximate it up to 80%.
Don't worry about colors much, but try to make the UI neat and clean.
If you find it difficult in picking color's go to this [Color picker](https://colorhunt.co/)

## DESIGN

![image](https://user-images.githubusercontent.com/83630220/135752606-32f95b40-a7b7-419e-aeb2-1475c62694dc.png)

**Comment by pri1311** [comment]
I believe the lower portion containing some image on the left can be not implemented? Right?

**Comment by rishikeshreddyakkireddy** [comment]
@pri1311  yes you can ignore it. are you interested to contribute to this issue cause I will assign if you are interested

**Comment by pri1311** [comment]
This design seems a little challenging to me honestly. Will you be uploading more designs? Or else I'll give this a shot.

**Comment by rishikeshreddyakkireddy** [comment]
I will be uploading more design's and also you can contribute to any of these 5 issues. I would suggest giving it a try. if you know grid flexbox and positioning in CSS that will be enough.

**Comment by pri1311** [comment]
Yep, please assign this to me.

**Comment by YashGupta29** [comment]
Hey, i would love to contribute. If the issue is not resolved till now. Can you please assign this to me?

**Comment by rishikeshreddyakkireddy** [comment]
@pri1311 thank you for letting me know, you are one of our best contributors you are always welcome here :)

**Comment by rishikeshreddyakkireddy** [comment]
@YashGupta29 are you interested to work on this ? I will assign it to you :)

**Comment by rishikeshreddyakkireddy** [comment]
@Qlio @YashGupta29 I will be assigning this issue to both of you and the best one will be merged
